src/components/OneTokenRequired.vue

```diff
@@ -0,0 +1,43 @@
+<template>
```

Comment:
I'm actually not understanding why there is 2 components for this?
They both seem to say the same thing as well with the difference being a "Buy gas" button